src/screens/Main.tsx

```diff
@@ -79,11 +79,7 @@ function Main({ className }: Props) {
   const [index, setIndex] = React.useState<number>(0);
 
   const searchItems = (choice: number) => {
-    for (let i = 0; i < items.length; i++) {
-      if (items[i].idx === choice) {
-        return items[i];
-      }
-    }
+    items.find(x => x.idx === choice)
```

Comment:
Sorry, too much rust. I guest `return` is misisng here.

Reply:
Was testing that and captured it (actually just the curly braces removed and it was fine)

Reply:
Still see the curlies in the diff?